lms/templates/support/contact_us.html

```diff
@@ -33,7 +33,7 @@
             'loginQuery': "${login_query() | n, js_escaped_string}",
             'dashboardUrl': "${reverse('dashboard') | n, js_escaped_string}",
             'homepageUrl': "${marketing_link('ROOT') | n, js_escaped_string}",
-            'submitFormUrl': "${reverse('ucsd_support_email') | n, js_escaped_string}",
```

Comment:
we had written our custom contact_us view. Now I have changed it back to its default view (zendesk_proxy_v1) .